.github/workflows/test.yml

```diff
@@ -114,10 +114,11 @@ jobs:
         if: github.event_name != 'schedule'
         run: shellcheck -o all -S style -s sh $(find . -iname "*.sh")
 
-  # This job reports the results of the matrixes above
+  # This job reports the results of the test jobs above and is used to enforce status checks in
+  # the repo settings without needing to update those settings every time the test jobs are updated.
   test:
     if: always()
-    needs: [clippy, test-matrix, all-features, oldstable]
+    needs: [rustfmt, clippy, test-matrix, oldstable, all-features, deno-checks, shellcheck]
```

Comment:
Understood...argument makes sense...this will be reverted.

We will need to be careful to ensure any future additions to the `Test` workflow are captured either in this list or in the branch protection settings for required status checks.